notes_app/client/app/notes/notes_service.js

```diff
@@ -32,6 +41,7 @@
       'create': function(note) {
         return $http.post(api_path, note).then(
           function successHandler(response) {
+            updateList(response.data.objects);
```

Comment:
It doesn't look right to me. It looks like you want to call `list` function to update `data.notes` from back-end